types/dev/dictionary-validate.d.ts

```diff
@@ -29,3 +29,8 @@ export type Schemas = {
     termBankV3: Schema;
     termMetaBankV3: Schema;
 };
+
+/**
+ * A map of regular expression identifiers of file types inside a dictionary and its corresponding schemas.
+ */
+export type SchemasDetails = [RegExp, unknown][];
```

Comment:
Name the array items.

Reply:
By name them i mean like `[something: RegExp, somethingElse: unknown][]`

Reply:
I totally forgot that was a thing in TS